src/gles_fns.rs

```diff
@@ -681,6 +681,45 @@ impl Gl for GlesFns {
         panic!("not supported");
     }
 
+    fn copy_image_sub_data(&self,
```

Comment:
This should probably be unsafe.

>   If the source and destination images are identical, and the source and destination rectangles overlap, the result of the operation is undefined.

Reply:
Okay. FWIW it looks like glBlitFramebuffers has an equivalent undefined behaviour, and we don't mark it as unsafe. Should I file a bug for that?

Can we assert that src_name != dst_name, and keep it safe? Or do we prefer to keep that sort of logic out of gleam?

Reply:
Yes, pointers of functions which failed to be loaded are set to a stub that panics.